src/validation/fields/nature-of-control.validation.ts

```diff
@@ -0,0 +1,14 @@
+import { checkFieldIfCheckboxTicked } from "../custom.validation";
+import { ErrorMessages } from "../error.messages";
+import { body } from "express-validator";
+
+export const nature_of_control_validations = [
+  body("")
```

Comment:
the href on the error message that appears in the box at the top of the screen doesn't work, and the noc checkboxes don't have red lines beside them when they fail validation. I've managed to do it here - https://github.com/companieshouse/overseas-entities-web/compare/roe-383-bo-other-legal-entity-validation

Reply:
Updated this.